gci/models.py

```diff
@@ -1 +1,117 @@
-# Create your models here.
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+from django.db import models
+
+
+class Student(models.Model):
+
+    identifier = models.IntegerField(primary_key=True)
+    display_name = models.CharField(max_length=100)
+    program_year = models.ManyToManyField('ProgramYear')
+
+    def __str__(self):
+        return self.display_name
+
+
+class Organization(models.Model):
+
+    identifier = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=100)
+    logo = models.ImageField(upload_to='', blank=True)
+    url = models.URLField(blank=True)
+    summary = models.TextField(max_length=300, default='')
+    students = models.ManyToManyField(Student)
```

Comment:
this is a derived attribute, not part of the data model.
organisations have task & instances, and students are attached to the instances.
a student becomes related to the org, only when a student claims a task.

however an organisation does have mentors, even if mentors are not associated with a task.